styleguide-content/stil-og-tone/stil-og-tone.md

```diff
@@ -0,0 +1,35 @@
+#### Vår måte å skrive og snakke på
+
+1. ##### Åpen
+
+   **Vi er ærlige - til å stole på**
+
+   Vi kommuniserer ærlig og med et reelt ønske om at kundene skal få mest mulig igjen for pengene sine.
+
+
+2. ##### Relevant
+
+   **Vi ser hverdagen fra kundens ståsted**
+
+   Vi snakker med kundene om deres økonomi - ikke bare om våre produkter. Lytt og bruk spørreord aktivt. Forsøk å bruke din innsikt om hva folk er opptatt av og del av vår kunnskap om hverdagsøkonomi.
+
+
+3. ##### Til stede hver dag
+
+   **Vi er til stede i hverdagen til kundene, og heier frem deres små seire og fremskritt**
+
+   Alle ønsker å bli sett og få gode tilbakemeldinger, også kunder og lokalsamfunn.
+
+
+4. ##### Tydelig
+
+   **Vi er modige, og fremstår som et tydelig alternativ i markedet**
+
+   Vi tør å være ubankske og å bruke det lokale og regionale til vår fordel når vi kommuniserer.
+
+
```

Comment:
This will probably generate five separate `ol`. See #71 for example.

Reply:
Thanks, fixed!